Src/FluentAssertions/TypeEnumerableExtensions.cs

```diff
@@ -77,5 +77,60 @@ public static IEnumerable<Type> ThatImplement<T>(this IEnumerable<Type> types)
         {
             return new TypeSelector(types).ThatImplement<T>();
         }
+
+        /// <summary>
+        /// Filters to only include types that are classes.
+        /// </summary>
+        public static IEnumerable<Type> ThatAreClasses(this IEnumerable<Type> types)
```

Comment:
None of the these new members are covered by tests.

Reply:
I'm a little bit confused since old members are also not covered with tests. 
Moreover, I see that all members of this class don't have any logic and they're just proxies to types that are covered by tests.
Are you sure they should be covered by tests? As I see such tests will inevitably be a copy-paste of other tests or such brittle things like Moqs `WasCalled` assertions

Reply:
You're right about the old code. We must have missed that in the past. But I spotted a bug in them during the review, which should have been caught by the tests.

Reply:
Got it. I'll think about how to make tests that will not be very brittle.

Reply:
Fixed